src/frontend/src/pages/PIADetailPage/messages.ts

```diff
@@ -0,0 +1,39 @@
+export default {
+  GeneralInfoSection: {
+    H2Text: {
+      en: `General Information`,
+    },
+    H2TextTwo: {
+      en: `Initiative details`,
+    },
+    H2TextThree: {
+      en: `Personal information`,
+    },
+  },
+  InitiativeDescriptionSection: {
+    H2Text: {
+      en: `What is the initiative?`,
+    },
+  },
+  InitiativeScopeSection: {
+    H2Text: {
+      en: `What is the scope of the PIA?`,
+    },
+  },
+  InitiativeDataElementsSection: {
+    H2Text: {
+      en: `What are the data or information elements involved in your initiative?`,
+    },
+  },
+  InitiativePISection: {
+    H2Text: {
+      en: `Did you list personal information in question 3?`,
+    },
+  },
+  InitiativeRiskReductionSection: {
+    H2Text: {
```

Comment:
please group messages wisely. Not sure if it is required here at all.

Also, there are discrepancies in the whole file where a text is not an H2, still called an `H2Text`.  If the message grouping is not required, we can always keep it to just one level as well